### PR TITLE
Updating CI env with 2022.03 chicagoland.pandemicresponsecommons.org 1647281018

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2022.03 to chicagoland.pandemicresponsecommons.org

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## Deployment changes
- awshelper
  - we will now be deploying a new service called argo-wrapper (uc-cdis/cloud-automation#1832)
  - Move datadog deployment to helm for easier configuration and upgrades (uc-cdis/cloud-automation#1824)
- fence
  - Requires a Fence DB migration (uc-cdis/fence#1003)
## Breaking changes
- sheepdog
  - Sheepdog now validates formats. If your dictionary uses formats and you have existing data or submission pipelines that do not respect these formats, this might be a breaking change (uc-cdis/sheepdog#363)

File: chicagoland.pandemicresponsecommons.org/manifest.json [hunk 1]
```diff
@@ -4,33 +4,33 @@
...
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:3.19.4",
-    "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:0.6.1",
+    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:2022.03",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2022.03",
+    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2022.03",
+    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2022.03",
+    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2022.03",
+    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2022.03",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2022.03",
+    "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2022.03",
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifest.json:21]: in QA you did not change the tube version https://github.com/uc-cdis/gitops-qa/pull/1907

Comment (atharvar28) [reply chicagoland.pandemicresponsecommons.org/manifest.json:21]: Yeah! the automation changes all the service versions, let me push a commit to revert back the tube version

**Review by paulineribeyre — DISMISSED** [review]

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## Deployment changes
- awshelper
  - we will now be deploying a new service called argo-wrapper (uc-cdis/cloud-automation#1832)
  - Move datadog deployment to helm for easier configuration and upgrades (uc-cdis/cloud-automation#1824)
- fence
  - Requires a Fence DB migration (uc-cdis/fence#1003)
## Breaking changes
- sheepdog
  - Sheepdog now validates formats. If your dictionary uses formats and you have existing data or submission pipelines that do not respect these formats, this might be a breaking change (uc-cdis/sheepdog#363)

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**